docs/topics/api/signing.rst

```diff
@@ -14,15 +14,34 @@ as multi-part formdata. This will create a pending version on the
 add-on and will prevent future submissions to this version unless
 validation or review fails.
 
-Example::
+If the upload succeeded then it will be submitted for
+validation and you will be able to check its status.
 
-    curl https://addons.mozilla.org/en-US/firefox/api/v3/addons/my-addon/versions/1.0/
-         -XPUT --form 'upload=@build/my-addon.xpi' -H 'Authorization: JWT <jwt-token>'
+.. http:put:: /en-US/firefox/api/v3/addons/(add-on-guid)/versions/(version)/
 
-The response will be the same as the :ref:`check-status` response.
+    **Request:**
 
-If your upload has the right metadata then it will be submitted for
-validation and you will be able to check its status.
+    .. sourcecode:: bash
+
+        curl https://addons.mozilla.org/en-US/firefox/api/v3/addons/my-addon/versions/1.0/
+            -XPUT --form 'upload=@build/my-addon.xpi' -H 'Authorization: JWT <jwt-token>'
+
+    :param addon-guid: the GUID for the add-on.
+    :param version: the version of the add-on.
+    :form upload: the add-on being uploaded.
+    :reqheader Content-Type: multipart/form-data
+
+    **Response:**
+
+    The response body will be the same as the :ref:`check-status` response.
```

Comment:
Do you know if these refs are global? I was originally thinking they were local to the file but I have a feeling they aren't maybe this should be ``:ref:`version-status` `` instead?